File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/DatasetVO.java
package com.sdkwork.app.model;

public class DatasetVO {
    private String label;
    private List<Object> data;
    private List<String> backgroundColor;
    private List<String> borderColor;
    private Integer borderWidth;

    public String getLabel() {
        return this.label;
    }
    
    public void setLabel(String label) {
        this.label = label;
    }

    public List<Object> getData() {
        return this.data;
    }
    
    public void setData(List<Object> data) {
        this.data = data;
    }

    public List<String> getBackgroundColor() {
        return this.backgroundColor;
    }
    
    public void setBackgroundColor(List<String> backgroundColor) {
        this.backgroundColor = backgroundColor;
    }

    public List<String> getBorderColor() {
        return this.borderColor;
    }
    
    public void setBorderColor(List<String> borderColor) {
        this.borderColor = borderColor;
    }

    public Integer getBorderWidth() {
        return this.borderWidth;
    }
    
    public void setBorderWidth(Integer borderWidth) {
        this.borderWidth = borderWidth;
    }
}

File: sdkwork-app-sdk-typescript/src/types/dataset-vo.ts
/** 数据集 */
export interface DatasetVO {
  /** 数据集名称 */
  label?: string;
  /** 数据值 */
  data?: unknown[];
  /** 背景颜色 */
  backgroundColor?: string[];
  /** 边框颜色 */
  borderColor?: string[];
  /** 边框宽度 */
  borderWidth?: number;
}
